dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -19,70 +19,83 @@ def to_honeybadger_context
 class ReportAbuseController < ApplicationController
   AGE_CUSTOM_FIELD_ID = 24_024_923
 
+  # Projects that are created by users with project validator permissions are
+  # blocked from abuse reports because they are created internally and we know
+  # they are safe. This reduces spamming of the report abuse feature.
+  def protected_project?
+    return false if params[:channel_id].blank?
+    owner_storage_id, _ = storage_decrypt_channel_id(params[:channel_id])
+    owner_user_id = user_storage_ids_table.where(id: owner_storage_id).first[:user_id]
+    project_owner = User.find(owner_user_id)
+    project_owner.permission?(UserPermission::PROJECT_VALIDATOR)
```

Comment:
looks better! now this looks right for projects owned by users, but may give an error for projects which do not appear in user_storage_ids, i.e. those created by signed-out users.